[CI] upgrade schemathesis to 4.1.0 from 3.39.15

davidxia opened the pull request:
It's better and faster.

https://github.com/schemathesis/schemathesis/releases/tag/v4.1.0
https://github.com/schemathesis/schemathesis/releases/tag/v4.0.0

Stranger6667 commented:
Just want to note that there could be some false positives in `positive_data_acceptance` and `negative_data_rejection` checks. I plan to solve them completely in `4.1` (scheduled for release this / next week)

davidxia committed:
[CI] upgrade schemathesis to 4.1.0 from 3.39.15

It's better and faster.

https://github.com/schemathesis/schemathesis/releases/tag/v4.1.0
https://github.com/schemathesis/schemathesis/releases/tag/v4.0.0

Signed-off-by: David Xia <david@davidxia.com>
